core/types/governance_new_market.go

```diff
@@ -214,7 +218,11 @@ func NewMarketConfigurationFromProto(p *vegapb.NewMarketConfiguration) *NewMarke
 	}
 	var liquidityMonitoring *LiquidityMonitoringParameters
 	if p.LiquidityMonitoringParameters != nil {
-		liquidityMonitoring = LiquidityMonitoringParametersFromProto(p.LiquidityMonitoringParameters)
+		var err error
+		liquidityMonitoring, err = LiquidityMonitoringParametersFromProto(p.LiquidityMonitoringParameters)
+		if err != nil {
+			return nil, fmt.Errorf("error getting new market configuration from proto: %s", err)
```

Comment:
Minor: should probably be `%w` to wrap the error.

Reply:
Addressed here https://github.com/vegaprotocol/vega/pull/7222/commits/935f9774a523f334d55e43742ff63338043eb2c4#diff-872de7073d1b2054247d4d9bfcdc63c49d530f5ee501083b4fc7ec970ce13051R467
Thanks!